src/dot_claude/commands/claude/generate-hook.md

```diff
@@ -0,0 +1,44 @@
+---
+name: claude:generate-hook
+description: Interactively generate a Claude hook configuration.
+---
+
+# Claude Hook Generator
+
+This command guides you through the process of creating a valid Claude hook configuration.
+
+## Instructions
+
+1.  **Engage the User**: Start a conversation to determine the necessary details for the hook. You must get the following information:
+    *   **Event Type**: The specific event that will trigger the hook (e.g., `PreToolUse`, `PostToolUse`).
+    *   **Matcher**: The tool(s) the hook should apply to (e.g., `Bash`, `Edit|Write`, `*`).
+    *   **Command**: The shell command that the hook will execute.
+
+2.  **Generate the Hook JSON**: Once you have all the required information, use the `generate-claude-hook` CLI tool to create the JSON configuration. The command will look like this:
+
+    ```bash
+    generate-claude-hook --event <EVENT_TYPE> --matcher "<MATCHER>" --command "<COMMAND>"
+    ```
+
+3.  **Present the JSON**: Display the generated JSON to the user for their review.
+
+4.  **Provide Merging Instructions**: Provide the user with the following `jq` command to merge the generated JSON into their settings file. You must explain what the command does and how to adapt it.
```

Comment:
This is a specific case

The goal is to specify the output, which might be a user json file to merge, project file to merge, local settings, or a new file entirely in here for potential usage in a later project install